### PR TITLE
UX: Chat browse redesign

### PR DESCRIPTION
**Pull request by chapoi**
This commit redesign the chat-channel-card on the channel browse page.
* tighter spacing
* removed settings button
* changed muted tag to icon
* different card layout
* removed joined tag 
* changed new-channel button to minimal link
* fixed search input cutoff 
* changed copy of tab to "Open" because "Opened" feels wrong

### Mobile Before – After
image

### Desktop Before – After
image

### CODE REVIEW

File: plugins/chat/assets/javascripts/discourse/components/chat-channel-card.gjs [hunk 1]
```diff
@@ -36,76 +36,59 @@ export default class ChatChannelCard extends Component {
...
-                @route="chat.channel.info.settings"
-                @models={{@channel.routeModels}}
-                class="chat-channel-card__tag -muted"
-                tabindex="-1"
-              >
-                {{i18n "chat.muted"}}
-              </LinkTo>
+              <span
+                class="chat-channel-card__muted"
+                aria-label="Channel is muted."
```
Comment (jjaffeux) [inline plugins/chat/assets/javascripts/discourse/components/chat-channel-card.gjs:42]: we an i18n locale here

Comment (jjaffeux) [reply plugins/chat/assets/javascripts/discourse/components/chat-channel-card.gjs:42]: Also do we need aria-label given we have a title`?

File: plugins/chat/assets/javascripts/discourse/components/chat/navbar/new-channel-button.gjs [hunk 2]
```diff
@@ -22,10 +22,7 @@ export default class ChatNavbarNewChannelButton extends Component {
         @action={{this.createChannel}}
         @icon="plus"
         @label={{if this.site.desktopView "chat.create_channel.title"}}
-        class={{concatClass
-          "c-navbar__new-channel-button"
-          (if this.site.mobileView "btn-flat")
-        }}
+        class={{concatClass "c-navbar__new-channel-button btn-transparent"}}
```
Comment (jjaffeux) [inline plugins/chat/assets/javascripts/discourse/components/chat/navbar/new-channel-button.gjs:25]:
```suggestion
        class="c-navbar__new-channel-button btn-transparent"
```

File: plugins/chat/assets/javascripts/discourse/components/chat-channel-card.gjs (hunk 1 above)
Comment (chapoi) [reply plugins/chat/assets/javascripts/discourse/components/chat-channel-card.gjs:42]: oh right forgot thx

File: plugins/chat/assets/javascripts/discourse/components/chat/navbar/new-channel-button.gjs (hunk 2 above)
Comment (jjaffeux) [reply plugins/chat/assets/javascripts/discourse/components/chat/navbar/new-channel-button.gjs]: That might cause a linting error if concatClass is used nowhere now (you will need to remove the import in this case)

File: plugins/chat/assets/javascripts/discourse/components/chat-channel-card.gjs (hunk 1 above)
Comment (chapoi) [reply plugins/chat/assets/javascripts/discourse/components/chat-channel-card.gjs:42]: > Also do we need aria-label given we have a title`?

Yes, aria-label for screenreaders, title for hover tooltip.

**Review by jjaffeux — APPROVED** [review]

**Pull request merged**